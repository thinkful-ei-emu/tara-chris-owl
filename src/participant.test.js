import React from 'react';
import ReactDOM from 'react-dom';
import Participant from './participant.js';
import renderer from 'react-test-renderer';

describe('Participant Component', () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<Participant key={'1'} name={'TommyThompson'} avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'} inSessionString="in-session" onStage={false}/>, div);

        ReactDOM.unmountComponentAtNode(div);
  })

  it('renders the UI as expected', () => {
      const tree = renderer
        .create(<Participant key={'1'} name={'TommyThompson'} avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'} inSessionString="in-session" onStage={false} />)
        .toJSON();
      expect(tree).toMatchSnapshot();  
      });
})

//key={'1'} name={'TommyThompson'} avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'} inSessionString="in-session" onStage={false}