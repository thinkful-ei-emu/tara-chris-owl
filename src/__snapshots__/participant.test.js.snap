// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant Component renders the UI as expected 1`] = `
<div
  className="participant"
>
  <img
    alt="TommyThompson's avatar"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
  <h3
    className="participantName"
  >
    TommyThompson
  </h3>
  <p
    className="in-session"
  >
    left session
  </p>
</div>
`;
